README.md

```diff
@@ -70,82 +116,135 @@ Docker is needed to run the project in a containerized environment. Follow the a
-ACCESS_SECRET_KEY=<your-secret-key>
-RESET_PASSWORD_SECRET_KEY=<your-secret-key>
-VERIFICATION_SECRET_KEY=<your-secret-key>
-OPENAPI_OUTPUT_FILE=../nextjs-frontend/openapi.json
-
-MAIL_USERNAME=<your-username>
-MAIL_PASSWORD=<your-password>
-MAIL_FROM=<your-from>
-MAIL_SERVER=<your-server>
+Copy the `.env.example` files to `.env` and update the variables with your own values.
+```bash
+cp project_name/.env.example project_name/.env
 ```
 
 **Frontend (`nextjs-frontend/.env.local`):**
-```env
-NEXT_PUBLIC_API_BASE_URL=http://localhost:8000
-OPENAPI_OUTPUT_FILE=openapi.json
+Copy the `.env.local.example` files to `.env.local` and update the variables with your own values.
+```bash
+cp project_name/.env.local.example project_name/.env.local
 ```
 
-### If you are not using Docker:
-To run the project locally, use the following commands:
-
-#### Backend
-1. Navigate to the `fastapi_backend` directory.
-2. Use Docker to run the database to avoid local installation issues. Build and start the database container:
+### Running the Database
+1. Use Docker to run the database to avoid local installation issues. Build and start the database container:
    ```bash
    docker compose build db
    docker compose up db
    ```
-3. Run the following command to apply database migrations:
+2. Run the following command to apply database migrations:
    ```bash
    make docker-migrate-db
    ```
-4. Start the FastAPI server:
+
+### If you are not using Docker:
+To setup the project environment locally, use the following commands:
+
+#### Backend
+
+1. Navigate to the `fastapi_backend` directory and run:
    ```bash
-   make start-backend
+   poetry install
    ```
 
 #### Frontend
-1. Navigate to the `nextjs-frontend` directory.
-2. Install dependencies (if not already installed):
+1. Navigate to the `nextjs-frontend` directory and run:
    ```bash
-   npm install
+   pnpm install
    ```
-3. Start the Next.js development server:
+
+### If you are using Docker:
+
+1. Build the backend and frontend containers:
+   ```bash
+   make docker-build-backend
+   make docker-build-frontend
+   ```
+
+## Running the Application
+
+If you are not using Docker:
+
+1. Start the FastAPI server:
+   ```bash
+   make start-backend
+   ```
+
+2. Start the Next.js development server:
    ```bash
    make start-frontend
    ```
 
-### If you are using Docker:
-1. Ensure Docker is running.
-2. Run the following command to build and start the backend, frontend, and database containers:
+If you are using Docker:
+1. Start the FastAPI server container:
    ```bash
    make docker-up-backend
+   ```
+2. Start the Next.js development server container:
+   ```bash
    make docker-up-frontend
    ```
-   This command will automatically set up the database and other necessary containers.
 
-3. To create the database schema for the first time, run:
+- **Backend**: Access the API at `http://localhost:8000`.
+- **Frontend**: Access the web application at `http://localhost:3000`.
+
+## Watchers
```

Comment:
Rename this section to "Hot Reload on development". This is the common name used for this, I checked.